From 55c43ee3158e5e83dcf51a7d6eaa35d4314c0833 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Mon, 21 Dec 2009 09:10:21 +1000
Subject: [PATCH] glsl: make stack compile.

it would be nice if this stuff would be test built on mesa's primary
platform with mesa's primary build system.

Signed-off-by: Dave Airlie <airlied@redhat.com>
---
 src/glsl/pp/sl_pp_macro.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/glsl/pp/sl_pp_macro.h b/src/glsl/pp/sl_pp_macro.h
index 2967baf..1d21068 100644
--- a/src/glsl/pp/sl_pp_macro.h
+++ b/src/glsl/pp/sl_pp_macro.h
@@ -33,6 +33,7 @@
 
 struct sl_pp_context;
 struct sl_pp_process_state;
+struct sl_pp_token_buffer;
 
 struct sl_pp_macro_formal_arg {
    int name;
-- 
1.6.5.2

